src/lib/features/project/project-owners-read-model.test.ts

```diff
@@ -0,0 +1,175 @@
+import dbInit, { type ITestDb } from '../../../test/e2e/helpers/database-init';
+import getLogger from '../../../test/fixtures/no-logger';
+import { type IUser, RoleName, type IGroup } from '../../types';
+import { randomId } from '../../util';
+import { ProjectOwnersReadModel } from './project-owners-read-model';
+
+describe('unit tests', () => {
+    test('maps owners to projects', () => {});
+    test('returns "system" when a project has no owners', async () => {
+        // this is a mapping test; not an integration test
+    });
+});
+
+let db: ITestDb;
+let readModel: ProjectOwnersReadModel;
+
+let ownerRoleId: number;
+let owner: IUser;
+let member: IUser;
+let group: IGroup;
+
+beforeAll(async () => {
+    db = await dbInit('project_owners_read_model_serial', getLogger);
+    readModel = new ProjectOwnersReadModel(db.rawDatabase, db.stores.roleStore);
+    ownerRoleId = (await db.stores.roleStore.getRoleByName(RoleName.OWNER)).id;
+
+    const ownerData = {
+        name: 'Owner User',
```

Comment:
Suggest changing this to "Owner Name" to be consistent with the member. (would also need to update the test on line 80.) But up to you 💁🏼 

```suggestion
        name: 'Owner Name',
```

Reply:
done in next PR